example_agents/papag/agent.py

```diff
@@ -69,11 +91,12 @@ def evaluate(
         cuda=False,
     ):
         num_processes = int(num_processes)
+        env_class, _ = self._get_env_class_and_kwargs()
         with self.PAPAGRun.evaluate_run(
             outer_run=active_component_run(self),
             model_input_run=self.model_input_run,
-            agent_identifier=self.__component__.identifier,
-            environment_identifier=self.__component__.identifier,
+            agent_identifier=self.Algo.__component__.identifier,
```

Comment:
We can discuss more if necessary, but in my PR against this PR (i.e. https://github.com/nickjalbert/agentos/pull/10) I revert this change to the agent_identifier because pointing at the algo component instead of the agent component makes it harder to differentiate the different "agents" that show up on the leaderboard. Specifically, when we publish a run of the sb3 agent using the PPO algo and a run of the PAPAG agent using the PPO algo, the table shows PPO in the "Agent" column for both runs.

Reply:
Yeah makes sense.  I think the trade-off here is that pointing the agent_identifier to the algo component is nice because then that component links directly to a PPO implementation in the papag or sb3 GitHub repos when you look at the component details in the web view.  

As it is, happy to accept the change for now and we can fiddle with this and/or the web views going forward.